admin/src/Components/MemberForm.js

```diff
@@ -96,7 +96,9 @@ const MemberForm = ({ member, onSave, onDelete }) => {
                     </div>
                 </fieldset>
 
-                {member.id && (
+                {!member.id ? (
+                    ""
```

Comment:
I think `null` is the recommended way to output nothing, not an empty string. Not sure if there are any differences in practice.